content/en/docs/how-tos/cluster-claim.md

```diff
@@ -3,62 +3,223 @@ title: "Creating a Cluster Pool"
+Pay attention to the following stanzas:
 
-It is suggested to put the relevant manifests to the cluster pool in the same `namespace`. Hive [replaces](https://github.com/openshift/hive/blob/master/docs/clusterpools.md#install-config-template) `baseDomain` with the value taken from the pool's spec and `metadata.name` with a random string in the installation config from `secret/install-config-aws-us-east-1` and then passes it onto the OpenShift installer. Customizations such as the number of workers and the type of instances can be made there. A `ClusterImageSet` which a cluster level CRD defines the production and its version of the installed cluster.
+1. `metadata.labels`: These labels will be used by jobs to specify what cluster they want to obtain for their testing.
+2. `spec.baseDomain`: A base domain for all clusters created in this pool. You usually create this domain yourself when
+   configuring the cloud platform account (see [prerequisites](#prerequisites)).
+3. `spec.imageSetRef`: A reference to a `ClusterImageSet` in the cluster that determines the exact version of clusters
+   created in the pool. `ClusterImageSets` are cluster-scoped resources and their manifests are present
+   in [`clusters/hive/pools`](https://github.com/openshift/release/tree/master/clusters/hive/pools) directory. Either
+   select one of the already available, or create a new one. DPTP maintains a set of `ClusterImageSets` that are
+   regularly bumped to most recent released OCP versions.
+4. `spec.installConfigSecretTemplateRef`: a reference to a `Secret` that serves as an installation config template. See
+   the [below section](#create-a-manifest-for-your-install-config-template-secret) for more information.
+5. `spec.platform.$CLOUD.credentialsSecretRef`: A reference to the secret you created in
+   the [Prepare your cloud platform credentials](#prepare-your-cloud-platform-credentials) section.
+6. `pullSecretRef.name`: Must be kept `pull-secret`. OpenShift CI will populate your namespace with this secret that
+   contains all pull secrets necessary to install an OCP cluster.
 
-If any manifest contains sensitive information, e.g., `secret/hive-aws-credentials`, see [how to add a secret to CI](/docs/how-tos/adding-a-new-secret-to-ci/). Make sure to set the key `secretsync/target-clusters: "hive"` for the secret to ensure it is synced only to the cluster that needs it.
+#### Sizing your cluster pool
```

Comment:
```suggestion
### Sizing your cluster pool
```

Reply:
This is intentional, sizing the cluster is a subsection of `ClusterPool`